{
    "title": "TensorFlow is an end-to-end open source platform for machine learning. In affected versions an attacker can craft a TFLite model that would trigger a division by zero error in LSH [implementation](https://github.com/tensorflow/tensorflow/blob/149562d49faa709ea80df1d99fc41d005b81082a/tensorflow/lite/kernels/lsh_projection.cc#L118). We have patched the issue in GitHub commit 0575b640091680cfb70f4dd93e70658de43b94f9. The fix will be included in TensorFlow 2.6.0. We will also cherrypick thiscommit on TensorFlow 2.5.1, TensorFlow 2.4.3, and TensorFlow 2.3.4, as these are also affected and still in supported range.",
    "score": 5.5,
    "severity": "MEDIUM",
    "cwe_id": "CWE-369",
    "cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
    "cpe_match": [
        "cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*",
        "cpe:2.3:a:google:tensorflow:2.6.0:rc2:*:*:*:*:*:*",
        "cpe:2.3:a:google:tensorflow:2.6.0:rc1:*:*:*:*:*:*",
        "cpe:2.3:a:google:tensorflow:2.6.0:rc0:*:*:*:*:*:*",
        "cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*",
        "cpe:2.3:a:google:tensorflow:2.5.0:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "technique_id": "T1083",
            "confidence": "27.0",
            "technique_name": "File and Directory Discovery"
        },
        {
            "technique_id": "T1087",
            "confidence": "13.4",
            "technique_name": "Account Discovery"
        },
        {
            "technique_id": "T1219",
            "confidence": "10.9",
            "technique_name": "Remote Access Software"
        }
    ]
}
